rules/less-is-more/rule.md

```diff
@@ -0,0 +1,61 @@
+---
+type: rule
+title: Do you know that Less is more ?
+uri: less-is-more
+authors:
+  - title: Josep Fernandez
+    url: https://www.ssw.com.au/people/joseph-fernandez
+created: 2023-02-22T04:20:12.183Z
+guid: 9d8ce87b-f835-48a2-b60c-5549466da209
+---
+A popular concept in product development and interphase design is the idea of “Less is more”.  But what do we actually mean by less is more? 
+
+Let’s start off by defining what it doesn’t mean:
+
+❌ It doesn’t mean that you work less on a project and be able to deliver more
+
+❌ It doesn’t mean we make decisions based on our own personal biases or perceptions of what we think is the easiest solution for the user
+
+❌ It does not mean we Include or develop a feature in a design or product just because we have the technical ability to do so
+
+✅ When we talk about "Less is More", we are talking about simplifying the way we present ideas to focus the user on making decisions about a task in the most effective way possible.
+
+Here are a few helpful ways we can employ a “Less is more” approach:
+
+## Leave it out:
+
+"Just because we can does not mean we should"
+
+Including a design element, feature or product enhancement, because we have the technical ability to do so with no impact on the user experience. It could overcomplicate a simple task, cause cognitive overload and ultimately distract an audience from processing through a task with less friction.  It’s Equally as important to leave out a good idea than it is to include one, especially if it doesn’t help the user achieve their goals.
+
+![Figure: ❌ Bad example - overbranded background](microsoftteams-image-withlogos.png "Figure: Bad")
```

Comment:
Would these images of the background be better without all the audience heads? Right now both images look too cluttered...
I think it should be zoomed in to the area that matters.

1. Please update if agreed

Reply:
Done!